.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm-debug.log
 tmp
+.idea
```

Comment:
.idea should be excluded on global level, remove this line

Reply:
@oleg-andreyev, I added ".idea" here because I usually open this project in PhpStorm. I see no harm in having this line. 
For example Webpack repo has it too https://github.com/webpack/webpack/blob/master/.gitignore#L11

Reply:
Okay,